### PR TITLE
fix #1135 sender chat text color inside file

### LINKED ISSUE #1135: [Chat screen] Sender text color inside file is hard to see
### Actual result:
The color of sender text inside file block is hard to see

### Expected Result:
Change text color to darker color 

### Environment:
Android + iOS

### PR DESCRIPTION
**Pull request by huynguyennovem**

### CODE REVIEW

File: lib/utils/twacode.dart [hunk 1]
```diff
@@ -581,16 +581,18 @@ class TwacodeRenderer {
   List<InlineSpan> spans = [];
   List<String>? files = [];
   final Key key;
+  final bool isMyMessage;
 
   TwacodeRenderer({
     required this.key,
     required this.twacode,
     required this.files,
+    required this.isMyMessage,
```
Comment (EvGen94) [inline lib/utils/twacode.dart:590]: I think It's not needed, we can use parentStyle

File: lib/utils/twacode.dart [hunk 2]
```diff
@@ -752,7 +754,7 @@ class TwacodeRenderer {
         child: Column(
           children: fileIds
               .map((id) => (id.isNotEmpty)
-              ? FileTile(fileId: id)
+              ? FileTile(fileId: id, isMyMessage: isMyMessage ?? false)
```
Comment (EvGen94) [inline lib/utils/twacode.dart:757]: isMyMessage: parentStyle.color == Colors.black ? false : true,

Comment (huynguyennovem) [reply lib/utils/twacode.dart:757]: You're right! Agreed with that

Comment (huynguyennovem) [reply lib/utils/twacode.dart:757]: You're right. We can handle it by this solution, even though it's similar to `hard code` (we are fixed the color checking by `Colors.black`). Anyway, I prefer dynamic parameter

**Review by EvGen94 — APPROVED** [review]

**Pull request merged**